package.json

```diff
@@ -27,13 +27,15 @@
     "find-rss": "^1.6.4",
     "glob": "^7.1.2",
     "graphql-client": "^2.0.0",
+    "iconv-lite": "^0.4.24",
```

Comment:
this should be moved into pnpmfile.js 

which dependency has a dependency on iconv-lite ?